newIDE/app/package.json

```diff
@@ -64,7 +62,6 @@
     "react-dnd": "7.7.0",
     "react-dnd-html5-backend": "7.7.0",
     "react-dnd-multi-backend": "3.2.2",
-    "react-dnd-touch-backend": "0.8.3",
```

Comment:
I think this is needed to have a stable version of this package. You can see that it has not been removed from the package-lock.json, I guess it's because another package is using it.

Reply:
done!